command/helper/config.go

```diff
@@ -172,6 +174,11 @@ func (c *Config) BuildConfig() (*server.Config, error) {
 		conf.RestoreFile = &c.RestoreFile
 	}
 
+	// set block time if not default
+	if c.BlockTime != 2000 {
```

Comment:
Here it would also help if you extract the default value to a constant as Milos suggested 👍

Reply:
Default value moved to const 631515404cf370d1fc015938ea35da52ff9dbaaf